src/CountBase.ts

```diff
@@ -109,12 +115,14 @@ export function openCountBase(
               throw new Error(`Could not insert data. ${error}`)
             })
         }
-      },
-      query(
-        partition: string,
-        rangeStart: number = 0,
-        rangeEnd: number = rangeStart
-      ): Promise<any[]> {
+      })
+    },
+    query(
+      partition: string,
+      rangeStart: number = 0,
+      rangeEnd: number = rangeStart
+    ): Promise<any[]> {
+      return getConfig().then(configData => {
```

Comment:
What was wrong with getting the configuration just once when you open the database? This should save a round-trip to the file system on each query